### PR TITLE
[docs] Add proxy user guide

### PR DESCRIPTION
**Pull request by ikaven1024**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Add user guide document for proxy.

**Which issue(s) this PR fixes**:
Part of https://github.com/karmada-io/karmada/issues/2372

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by ikaven1024** [comment]
CC @RainbowMango @XiShanYongYe-Chang

File: docs/userguide/globalview/proxy-global-resource.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+Creating requests are redirected to karmada controller pannel, so this resources will be created in controller panel.
+
+### Can I read resources by selectors?
+
+Label selectors are fully supported. While field selectors are limited to `metadata.name` and `metadata.namespace`
+
+### When I get pods with kubectl, only NAME and AGE columns are displayed
+
+Yes, `kubeclt` use `application/json;as=Table;g=meta.k8s.io;v=v1 ` as `content-type`, while proxy only implement [`defaultTableConvertor`](https://github.com/karmada-io/karmada/blob/614e28508336d6c03a938ce1bf0678dafef034f0/vendor/k8s.io/apiserver/pkg/registry/rest/table.go#L38-L40) as TableConvertor.
```
Comment (RainbowMango) [inline docs/userguide/globalview/proxy-global-resource.md:68]:
```suggestion
Yes, `kubectl` use `application/json;as=Table;g=meta.k8s.io;v=v1 ` as `content-type`, while proxy only implement [`defaultTableConvertor`](https://github.com/karmada-io/karmada/blob/614e28508336d6c03a938ce1bf0678dafef034f0/vendor/k8s.io/apiserver/pkg/registry/rest/table.go#L38-L40) as TableConvertor.
```

File: docs/userguide/globalview/proxy-global-resource.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+### Step1: Obtain the karmada-apiserver Certificate
+
+For Karmada deployed using `hack/local-up-karmada.sh`, you can directly copy it from the `$HOME/.kube/` directory.
+
+```shell
+cp $HOME/.kube/karmada.config $HOME/karmada-proxy.config
+```
+
+### Step3: Access by proxy
```
Comment (XiShanYongYe-Chang) [inline docs/userguide/globalview/proxy-global-resource.md:25]: The steps seem to be messed up.

File: docs/userguide/globalview/proxy-global-resource.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
+---
+title: Proxy Global Resources
+---
+
+## Introduce
+
+The newly introduced [proxy](https://github.com/karmada-io/karmada/blob/master/docs/proposals/resource-aggregation-proxy/README.md) feature allows users to access all the resources both in karmada controller panel and member clusters. With it, users can:
+
+- create, update and delete resources in controller panel, such as deployments and jobs. All the request behaviors are supported, just like using `karmada-apiserver`.
+- update and delete resources in member clusters, such as pods, nodes, and customer resources.
```
[the remark this answers is not in the document]
Comment (ikaven1024) [reply docs/userguide/globalview/proxy-global-resource.md:10]: > Can we not create resources in a member cluster?

`Create` is not supported, as i explain in FAQ

**Review by XiShanYongYe-Chang — COMMENTED** [review]
LGTM, thanks~

**Review by RainbowMango — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**